helpers/build_packages.sh

```diff
@@ -194,6 +196,7 @@ if [ "$BUILDMW" = "1" ]; then
             # pull device's bluez4 configs correctly
             sb2 -t $VENDOR-$DEVICE-$PORT_ARCH -m sdk-install -R zypper remove bluez-configs-mer
         fi
+        BUILD_INDIVIDUAL_ASK=n
```

Comment:
code readability suffers with such approach

Reply:
I also wondered why not just use BUILD_ALL

Reply:
I thought it would be better to set that questioning is needed than set when its not needed and renamed the variable to reflect that.

Reply:
I don't think setting this new flag to =y and to =n is done in the right places. Will it ask or not, in cases of:
- `build_packages.sh`
- `build_packages.sh --mw`
- `build_packages.sh --mw=libhybris`

?

Reply:
The idea was that when $BUILDMW is 1 that the script asks if each pkg should be build, in every other case it doesn't.

Reply:
Why is the flag set to =n (`BUILD_INDIVIDUAL_ASK=n`) in such place where it's never read by any line of code again after that?

Reply:
Because I oversaw that and made an error. Fixed. Thanks for pointing out.